src/pipelines/sentence_embeddings/mod.rs

```diff
@@ -0,0 +1,21 @@
+pub mod builder;
+mod config;
+pub mod layers;
+mod pipeline;
+
+pub use builder::SentenceEmbeddingsBuilder;
+pub use config::{
+    SentenceEmbeddingsConfig, SentenceEmbeddingsModule, SentenceEmbeddingsModuleType,
+    SentenceEmbeddingsModules, SentenceEmbeddingsTokenizerConfig,
+};
+pub use pipeline::{
+    SentenceEmbeddingsModel, SentenceEmbeddingsModelOuput, SentenceEmbeddingsOption,
+    SentenceEmbeddingsTokenizerOuput,
+};
+
+pub type Attention = Vec<f32>; // Length = sequence length
```

Comment:
If the attention output is not included in the Sentence Embeddings output (see other comments), these types could be removed.

Reply:
Sure, let me know what you decide.